src/actions/likes.actions.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+
+import likeConstants from '../constants/likes.constants';
+
+export const likeSuccess = count => ({
+  type: likeConstants.LIKES_SUCESS,
+  count: count.response,
+});
+
+export const dislikeSuccess = count => ({
+  type: likeConstants.DISLIKE_SUCESS,
+  count: count.response,
+});
+
+const token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1NDUzMTQ4MTMsImlhdCI6MTU0NDcxMDAxMywiaWQiOjUxfQ.qB3oqQNnoeoYnaTRIp9h4TXAmARQLfUjVVZzUpjAfCw';
```

Comment:
Kindly pick the token from the logged in user and this pr will be good to go..

Reply:
This has been resolved.